src/main/java/com/quizapp/enterprise/persistence/GameTracker.java

```diff
@@ -28,18 +28,21 @@ public void addGame(Game game){
     }
 
     public Game getGameByCode(String gameCode){
-        return games.stream().filter(g -> g.getGameCode().equals(gameCode)).findAny().get();
+        return games.stream()
+                .filter(g -> g.getGameCode().equals(gameCode))
+                .findAny()
+                .get();
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3